terraform/cloudfoundry/cf_api_security_groups.tf

```diff
@@ -17,7 +17,7 @@ resource "aws_security_group" "cf_api_elb" {
     to_port   = 443
     protocol  = "tcp"
     cidr_blocks = [
-      "${split(",", var.office_cidrs)}",
```

Comment:
I think we should keep office cirds here. Have office cirds defined in one place only (this variable) and the tennant_cidrs would only contain additional IPs to enable